packages/esbuild/README.md

```diff
@@ -0,0 +1,140 @@
+# ESbuild 
+Esbuild is a module bundler and minifier for JavaScript and CSS.
```

Comment:
While this _might_ work with CSS, we don't use that here. We just use it to bundle JS and TS :slightly_smiling_face: